mad-dog/src/madDog.js

```diff
@@ -56,31 +56,37 @@ class MadDog {
     }, TICK_INTERVAL_SEC * 1000)
   }
 
-  stop() {
+  stop () {
     clearInterval(this.tickToken)
-    if (this.netemProcess) this.netemProcess.kill()
+    if (this.netemProcess) {
+      // Kill original process
+      this.netemProcess.kill()
+      // Set packet loss to 0% aka revert back to normal
+      exec(`pumba netem --tc-image gaiadocker/iproute2 -d ${this.networkDownDurationSec}s loss -p 0 containers ${this.netemProcessService}`)
+    }
     if (this.pauseProcess) this.pauseProcess.kill()
   }
 
-  _setNetworkConditions(serviceName) {
+  _setNetworkConditions (serviceName) {
     const service = serviceName || _.sample(this.services)
     logger.info(
       `Setting ${this.packetLossPercent}% packet loss for service: ${service}`
     )
+    this.netemProcessService = service
     this.netemProcess = exec(`pumba netem --tc-image gaiadocker/iproute2 -d ${this.networkDownDurationSec}s loss -p ${this.packetLossPercent} containers ${service}`)
     this._setProcessLogs(this.netemProcess)
   }
 
-  _setProcessLogs(process) {
+  _setProcessLogs (process) {
     process.stdout.on('data', (data) => logger.info(`stdout: ${data}`))
     process.stderr.on('data', (data) => logger.error(`stderr: ${data}`))
     process.on('exit', (code) => {
-      logger.info(`child process exited with code ${code}`)
+      if (code) logger.info(`child process exited with code ${code}`)
       process = null
     })
   }
 
-  _pauseService(serviceName) {
+  _pauseService (serviceName) {
     const service = serviceName || _.sample(this.services)
     didPause = true
     logger.info(`Pausing service: [${service}]`)
```

Comment:
there's no way line 93 below works - `containers` is misspelled. i wonder if that is causing any of test weirdness you're seeing?

Reply:
o whoa good catch. let me update this and rerun!!

Reply:
oh actually, I don't think we run the pause command for the reconfig tests lol but good catch!